docs/live_update_reference.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Live Update Reference
+layout: docs
+---
+##### (This doc provides the technical specifications of Tilt's `LiveUpdate` functionality. For a tutorial that walks you through a sample project, see [Optimizing a Tiltfile](live_update_tutorial.html).)
```

Comment:
if we're calling this a technical spec, I'd be inclined to:
1. make at least the first instance of each method a hyperlink to the api doc
2. mention the run trigger behavior somewhere (define below's use of "relevant")